src/traversals/dfs.jl

```diff
@@ -5,12 +5,37 @@
     is_cyclic(g)
 
 Return `true` if graph `g` contains a cycle.
-
+  
```

Comment:
Ah sorry about that, in that comment here I meant that you should remove the whitespace, but there should still be a newline here, otherwise the `### Implementation Notes` below will not render correctly.